Documentation/help_files/mfa_help.php

```diff
@@ -0,0 +1,319 @@
+<?php
+/**
+ * Message Center Help.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2019 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ 
+use OpenEMR\Core\Header;
+
+require_once("../../interface/globals.php");
+?>
+<!DOCTYPE HTML>
+<html>
+    <head>
+    <?php Header::setupHeader();?>
+    <title><?php echo xlt("Multi Factor Authorization Help");?></title>
+    <style>
+        .oe-help-add-info{
+            padding:15px;
+            border:6px solid;
+            font-style: italic;
+        }
+    </style>
+    </head>
+    <body>
+        <div class="container oe-help-container">
+            <div>
+                <center><h2><a name='entire_doc'><?php echo xlt("Multi Factor Authorization Help");?></a></h2></center>
+            </div>
+            <div class= "row">
+                <div class="col-sm-12">
+                    <div>
+                    <p><?php echo xlt("When dealing with protected health information (PHI) and  personally identifying information (PII) it is important to allow only authorized users access to the data");?>.
```

Comment:
make sure you don't have double spacing in constants, or it will cause problems in the translation pipeline

Reply:
will do

Reply:
fixed